src/Kiota.Builder/KiotaBuilder.cs

```diff
@@ -234,7 +234,7 @@ private void CreateRequestBuilderClass(CodeNamespace currentNamespace, OpenApiUr
         else
         {
             var targetNS = currentNode.DoesNodeBelongToItemSubnamespace() ? currentNamespace.EnsureItemNamespace() : currentNamespace;
-            var className = currentNode.GetClassName(requestBuilderSuffix);
+            var className = currentNode.DoesNodeBelongToItemSubnamespace() ? currentNode.GetClassName("Item" + requestBuilderSuffix) :currentNode.GetClassName(requestBuilderSuffix);
```

Comment:
please move this Item prefix to a constant

Reply:
Resolved in latest update.